src/options/view.rs

```diff
@@ -283,16 +276,16 @@ impl TimeFormat {
             "iso"      => Ok(Self::ISOFormat),
             "long-iso" => Ok(Self::LongISO),
             "full-iso" => Ok(Self::FullISO),
-            _ => Err(OptionsError::BadArgument(&flags::TIME_STYLE, word))
+            _ => Err(OptionsError::BadArgument("Time_Style".to_string(), word.to_string_lossy().to_string()))
```

Comment:
Not sure why you changed `w.to_os_string()` to `w.clone()`, since then you `.to_string_lossy()` everytime you use the variable instead. I think it was fine before the change